packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -66,16 +66,16 @@ export class Breadcrumb extends BaseComponent<IBreadcrumbProps, IBreadcrumbState
   }
 
   public render() {
-    let { className } = this.props;
+    let { className, navigationLandmarkAriaLabel } = this.props;
```

Comment:
@cschleiden @jspurlin are you ok with the name of this prop? Or should it be just `ariaLabel`? I want to make sure we have a consistent api surface.